docs/guides/guide-delete-or-replace-metrics.md

```diff
@@ -0,0 +1,299 @@
+    "instance": "node-exporter:9100"
+  },
+  "values": [
+    33604390912,
+    33604390912,
+    33604390912,
+    33604390912
+  ],
+  "timestamps": [
+    1656669031378,
+    1656669032378,
+    1656669033378,
+    1656669034378
+  ]
+}
+
+```
+
+In this example, we will replace the values of `node_memory_MemTotal_bytes` from `33604390912` to `17179869184` (from 32Gb to 16Gb) via [sed](https://linux.die.net/man/1/sed), but it can be done in any of the available ways.
+
+```console
+sed -i 's/33604390912/17179869184/g' data.jsonl
+```
+
+Let's check the changes in data.jsonl with `cat`:
+
+```console
+cat data.jsonl | jq
+```
+
+The expected output will be the next:
+
+```json
+{
+  "metric": {
+    "__name__": "node_memory_MemTotal_bytes",
+    "job": "hostname.com",
+    "instance": "node-exporter:9100"
+  },
+  "values": [
+    17179869184,
+    17179869184,
+    17179869184,
+    17179869184
+  ],
+  "timestamps": [
+    1656669031378,
+    1656669032378,
+    1656669033378,
+    1656669034378
+  ]
+}
+```
+
+### Delete metrics
+
+See [How-to-delete-metrics](https://docs.victoriametrics.com/guides/guide-delete-or-replace-metrics.html#how-to-delete-metrics) from the previous paragraph
+
+### Import metrics
+
+Victoriametrics supports a lot of [ingestion protocols](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-time-series-data) and we will use [import from JSON line format](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-data-in-json-line-format).
+
+The next command will import metrics from `data.jsonl` to VictoriaMetrics:
+
+**Single-node VictoriaMetrics**:
+
+<div class="with-copy" markdown="1">
+
+```console
+curl -v -X POST http://127.0.0.1:8428/api/v1/import -T data.jsonl
+```
+</div>
+
+**Cluster version of VictoriaMetrics**:
+
+<div class="with-copy" markdown="1">
+
+```console
+curl -v -X POST http://127.0.0.1:8480/insert/0/prometheus/api/v1/import -T data.jsonl
+```
+</div>
+
+### Check imported metrics
+
+**Single-node VictoriaMetrics**:
+
+<div class="with-copy" markdown="1">
+
+```console
+curl -X POST -g 'http://127.0.0.1:8428/api/v1/export -d match[]=node_memory_MemTotal_bytes
+```
+
+</div>
+
+**Cluster version of VictoriaMetrics**:
+
+<div class="with-copy" markdown="1">
+
+```console
+curl -X POST -g http://127.0.0.1:8481/select/0/prometheus/api/v1/export -d match[]=node_memory_MemTotal_bytes
+```
+
+</div>
+
+The expected output will look like:
+
+```json
+{
+  "metric": {
+    "__name__": "node_memory_MemTotal_bytes",
+    "job": "hostname.com",
+    "instance": "node-exporter:9100"
+  },
+  "values": [
+    17179869184,
+    17179869184,
+    17179869184,
+    17179869184
+  ],
+  "timestamps": [
+    1656669031378,
+    1656669032378,
+    1656669033378,
+    1656669034378
+  ]
+}
+```
+
+## Summary
+
+* We have learned how to remove metrics from VictoriaMetrics
+* We have learned how to update(replace) metrics in VictoriaMetrics
```

Comment:
```suggestion
```

Reply:
This was resolved with no changes. Is this intentional?

Reply:
> This was resolved with no changes. Is this intentional?

Nope, sorry.